source/includes/usage-examples/code-snippets/count.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+func main() {
+
+	var uri string
+	if uri = os.Getenv("MONGODB_URI"); uri == "" {
+		log.Fatal("You must set your `MONGODB_URI' environmental variable. See\n\t https://docs.mongodb.com/drivers/go/current/usage-examples/")
```

Comment:
Any reason for the `\n\t`?
```suggestion
		log.Fatal("You must set your 'MONGODB_URI' environmental variable. See\n\t https://docs.mongodb.com/drivers/go/current/usage-examples/")
```

Reply:
Also, https://docs.mongodb.com/drivers/go/current/usage-examples/ doesn't seem to exist.

Reply:
I believe `\n\t `was included for it to be more visually appealing. And yes, the page doesn't exist at the moment, but once deployed, it will be the [usage example landing page](https://github.com/mongodb/docs-golang/pull/22).